chrome.manifest

```diff
@@ -0,0 +1,5 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I don't think we need to put license blocks at the top of our files, since we have a repo-wide license file.